### ISSUE TITLE
Code Review Discussion

### ISSUE THREAD

**Issue by smudge202** [danielcirket/SimpleContainer#1]
To better keep ariugwu/mudz#4 on track, creating an issue on your repo, @danielcirket . How far do you want me to dig and/or what kind of information do you feel would be useful to you?

**Comment by danielcirket** [comment]
Anything at all that you feel is constructive - good or bad, I'm all for any feedback, as it all helps.

**Comment by smudge202** [comment]
Fair enough. I've been talking in some streams about possibly doing some twitch streaming in which I code review people's work, provide some tips and explain various aspects of code. Wondering if this might be a good starting point? That the code is open source I'm assuming you don't mind people seeing it, but would you be adverse to my streaming a code review later this evening (thinking about 22:00 BST/GMT+1 for about an hour) where I'll go through the code, raise issues for things I don't think are quite right, provide some tips, and hopefully manage to PR some suggestions?

**Comment by smudge202** [comment]
Make that 21:00. I don't want to miss @ariugwu's stream! :)

**Comment by danielcirket** [comment]
Yeah that's not a problem, I had my object mapper (micro ORM) critiqued on @whilke's stream, so reviewing this on stream is all OK with me! :)

**Comment by smudge202** [comment]
Looking to start in about 30 minutes.

You're welcome to join me on stream by joining our public teamspeak, if you'd like to talk through any of the issues.

Teamspeak details are on the twitch page: http://www.twitch.tv/smudge202

I may not be on there just yet, lots to sort out before I get started!

**Comment by smudge202** [comment]
So I think we got off to a pretty slow but good start last night. I'll do some work when I get home to get my fork all caught up and any other admin that needs doing. Then crack-on with part 2 of the stream at 21:00 tonight. The first part can be found here: https://www.youtube.com/watch?v=bm82yPn7AUY

**Comment by danielcirket** [comment]
Awesome, I gave your channel a subscribe after arigwu's stream so I see it in my feed this morning.
